sqlmesh/dbt/manifest.py

```diff
@@ -270,6 +267,33 @@ def _load_profile(self) -> Profile:
             target_override=self.target.name,
         )
 
+    def _disabled_ref(self, ref: str) -> bool:
+        if not self._disabled_refs:
```

Comment:
if self._disabled_refs is None:
    for ... 

if you have no disabled refs, you're going to be constantly reloading the manifest with this current code